source/extensions/filters/http/lua/wrappers_reason_phrase.cc

```diff
@@ -0,0 +1,43 @@
+#include "source/extensions/filters/http/lua/wrappers.h"
+
+#include "source/common/http/header_map_impl.h"
+#include "source/common/http/header_utility.h"
+#include "source/common/http/utility.h"
+#include "source/extensions/filters/common/lua/wrappers.h"
+#include "source/extensions/http/header_formatters/preserve_case/preserve_case_formatter.h"
+#include "envoy/http/header_formatter.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Lua {
+
+int HeaderMapWrapper::luaSetHttp1ReasonPhrase(lua_State* state) {
```

Comment:
nit: can you just add this to the other file? Not sure this needs a new file when it's part of the header map wrapper?

Reply:
Yup, will do! The separate file was mostly to reduce upstream merge conflicts if we had to maintain this feature in our own branch.